Makefile

```diff
@@ -200,27 +127,14 @@ clean_python_pb:
 cleanall : clean
 	-@rm -rf proto/*.pb.c proto/*.pb.h
 
-load: all
-	python -m ledgerblue.loadApp $(APP_LOAD_PARAMS)
-
-load-offline: all
-	python -m ledgerblue.loadApp $(APP_LOAD_PARAMS) --offline
-
-delete:
-	python -m ledgerblue.deleteApp $(COMMON_DELETE_PARAMS)
-
-# import generic rules from the sdk
-include $(BOLOS_SDK)/Makefile.rules
-
 #add dependency on custom makefile filename
 dep/%.d: %.c Makefile
 
-listvariants:
-	@echo VARIANTS COIN hedera
-
 check:
 	@ clang-tidy \
 		$(foreach path, $(APP_SOURCE_PATH), $(shell find $(path) -name "*.c" -and -not -name "pb*" -and -not -name "glyphs*")) -- \
```

Comment:
Is it needed? Used?
Should it rather be a global directive, part of the SDK in that case?

Reply:
Something added by the developers originally, don't know if anyone uses it anymore.